### ISSUE TITLE
Final Peer Review - pg334

### ISSUE THREAD

**Issue by pgokhale2000** [Ting-Kerbell/Success_of_Music#10]
The goal of this project is to use supervised machine learning to predict the commercial success of songs using auditory features to understand what makes music popular.

Things Done Well:
- This group did well in their means of gathering data. Using Spotify along with 3 popular charts is probably the best source to get data for a project on music hits. Furthermore, addressing the discrepancies by utilizing Spotify's API and Search function definitely leads to more accurate results. 
- The graphs and charts included to visualize the data were definitely helpful. This included correlation maps, violin plots, and several bar charts to analyze the features. This helped show exactly which features were the most important (ie energy) and which loss function (logistic loss) was the best to use.
- The variety of models and tuning techniques used allowed the results to be more accurate. The report talks about the use of the XGBoost Model, SVM, K-Fold Cross Validation, grid search, regularization, and up/downsampling. This is great because it helps with reducing overfitting while still improving accuracy. 

Areas of Improvement:
- It would have been helpful for the report to mention whether fairness was an important criterion in choosing a model for the application. Although the models were well explained in terms of increasing accuracy, fairness and possibility of protected attributes is crucial to consider as well.
- It could be made more clear what the real life applications of this project could be. It mentions at the beginning that it could help artists know what to put into their songs to make them hits, but towards the end of the report, correctly identifies that doing so would create a weapon of math destruction as this could ruin the integrity and creativity of the music industry. Are there better applications of this project then?
- All of the data is from the years 1999-2019, and using a span of two decades could skew the results by a lot. The report does mention that once the internet became popular the model became less accurate as other factors became more heavily involved, but in general it may be a bit unfair to combine all of this data. 20 years is a long time and the music industry changes quite frequently. Perhaps it would be better to split up the decades so that the model could provide better results from one particular decade.
